In 8.0.0, finders were refactored and moved into a different place, resulting
in build breakage when building the new version with the old one in the CBE.

https://github.com/pypa/setuptools_scm/commit/3b5b8a7cb3ec62c7a655d51ee272f08a1c44034b

This workaround that can be removed once the new version gets into the CBE.

--- setuptools_scm-8.1.0/src/setuptools_scm/_file_finders/__init__.py
+++ setuptools_scm-8.1.0/src/setuptools_scm/_file_finders/__init__.py
@@ -106,6 +106,9 @@ def find_files(path: _t.PathT = "") -> l
         iter_entry_points("setuptools_scm.files_command"),
         iter_entry_points("setuptools_scm.files_command_fallback"),
     ):
+        # don't look for finders in the old place (temporary workaround)
+        if ep.value.startswith("setuptools_scm.file_finder_"):
+            continue
         command: Callable[[_t.PathT], list[str]] = ep.load()
         res: list[str] = command(path)
         if res:
